src/graphql/elasticsearch/attribute/resolver.js

```diff
@@ -3,21 +3,21 @@ import client from '../client';
 import { buildQuery } from '../queryBuilder';
 import { getIndexName } from '../mapping'
 
-async function listAttributes (attributes, context, rootValue, _sourceInclude) {
+async function listAttributes (attributes, context, rootValue, _source_include) {
   let query = buildQuery({ filter: attributes, pageSize: 150, type: 'attribute' });
 
-  if (_sourceInclude === undefined) {
-    _sourceInclude = config.entities.attribute.includeFields
+  if (_source_include === undefined) {
```

Comment:
`typeof _source_include === undefined` or just `if (!_source_include)` is enough

Reply:
Changed it.